index.d.ts

```diff
@@ -1399,6 +1406,7 @@ declare namespace Eris {
   }
   interface WebhookPayload {
     allowedMentions?: AllowedMentions;
+    attachments: PartialAttachment[];
```

Comment:
```suggestion
    attachments?: PartialAttachment[];
```
This should be optional